dnf/repo.py

```diff
@@ -459,7 +459,7 @@ class Repo(dnf.conf.RepoConf):
     def __init__(self, name=None, parent_conf=None):
         # :api
         super(Repo, self).__init__(section=name, parent=parent_conf)
-        self._repofile = None
+        self.repofile = None
```

Comment:
Yes, I know, but I just added it to '__init__' so it is clearer to know which attributes Repo has (I spent some time looking for it). When I renamed it to '_repofile' it broke dnf-plugins/config_manager.py, that's why I renamed it back to 'repofile'. So should I leave it or create cosmetic PR for dnf-plugins?

>